src/Entity/Dataset.php

```diff
@@ -569,26 +571,7 @@ public function getIssueTrackingTicket()
      */
     public function getCitation()
```

Comment:
Publink application uses the Dataset GET Citation API, which uses this method from the Entity. So changing this would change the behavior of publink